sdk/servicebus/azure-servicebus/tests/test_sb_client.py

```diff
@@ -477,3 +477,58 @@ def test_backoff_fixed_retry(self):
         # time.sleep() in _backoff will take AT LEAST time 'exp' for retry_mode='exponential'
         # check that fixed is less than 'exp'
         assert sleep_time_fixed < backoff * (2 ** 1)
+
+    @pytest.mark.liveTest
+    @pytest.mark.live_test_only
+    @CachedResourceGroupPreparer(name_prefix='servicebustest')
+    @CachedServiceBusNamespacePreparer(name_prefix='servicebustest')
+    def test_custom_client_id_sender(self, servicebus_namespace_connection_string):
```

Comment:
for sake of thoroughness, might want to add the following:
* async tests
* topic_sender, subscription_receiver tests